wallet/src/signer/trezor_signer/mod.rs

```diff
@@ -211,17 +219,21 @@ impl TrezorSigner {
         operation(&mut client).map_err(|e| TrezorError::DeviceError(e.to_string()).into())
     }
 
-    fn make_signature<F>(
+    #[allow(clippy::too_many_arguments)]
+    fn make_signature<'a, 'b, F, F2>(
         &self,
         signatures: &[MintlayerSignature],
-        destination: &Destination,
+        standalone_inputs: Option<&'a [StandaloneInput]>,
```

Comment:
Why pass an `Option` here? `None` can be represented via an empty slice.

Currently you have the technical possibility of passing `Some(&[])` which doesn't make much sense. Also, it makes the lines `Some(_) => return Err(TrezorError::MultisigSignatureReturned.into())` technically incorrect, because `MultisigSignatureReturned` isn't a correct error type in the case of an empty slice.